src/modules/Elsa.Studio.Workflows/Components/WorkflowDefinitionEditor/Components/ActivityProperties/Tabs/PersistenceTab.razor

```diff
@@ -0,0 +1,73 @@
+﻿@using Elsa.Api.Client.Resources.ActivityDescriptors.Models
+@using Elsa.Api.Client.Resources.WorkflowDefinitions.Enums
+@using Variant = MudBlazor.Variant
+@inherits StudioComponentBase
+
+
+@{
+    
+    RenderFragment GetSelectComponent2(String propertyName, IDictionary<string, LogPersistenceMode> properties) =>

```

Comment:
Perhaps we can rename this to something else than `GetSelectComponent2`?

Reply:
Good catch, I've renamed it to `GetSelectComponent`